.studiorc

```diff
@@ -140,10 +140,14 @@ _start-builder() {
     start-builder originsrv
     start-builder sessionsrv
     start-builder worker
+    start-builder minio
     sleep 2
     generate_bldr_keys
     upload_github_keys
     echo "Builder Started: Navigate to http://localhost/#/pkgs/core to access the web UI."
+    echo "Minio login available at http://localhost:9000"
+    echo "Username: AWSKEYIDIBELIEVE"
```

Comment:
Minor - would be nice to default the key/secret to something simple and easy to remember/use, like "dev" / "dev".